DangerSwap/DangerSwap/Services/Scrapper/ScrapperService.cs

```diff
@@ -15,7 +15,7 @@ public class ScrapperService
         private readonly string _cryptoRatesFileName = string.Empty;
         private readonly string _fiatRatesFileName = string.Empty;
```

Comment:
11 - 16 row. Redundant initialization, values will be assigned in the constructor.